modules/Router.js

```diff
@@ -62,6 +62,8 @@ const Router = React.createClass({
       if (error) {
         this.handleError(error)
       } else {
+        router.location = state.location
+        router.params = state.params
```

Comment:
Can you clarify how this related to this PR? It only puts some stuff onto `this.context.router` so it is not related to `react-router-redux` or its actions.